examples/specs/bubble_health_income.json

```diff
@@ -0,0 +1,23 @@
+{
+  "description": "A bubble plot showing the correlation between health and income for 187 countries in the world.",
```

Comment:
Actually we copied this from that article, right?  If so, please credit the article.